go/protocol/chat1/notify.go

```diff
@@ -215,6 +215,11 @@ type ChatTLFFinalizeArg struct {
 	FinalizeInfo ConversationFinalizeInfo `codec:"finalizeInfo" json:"finalizeInfo"`
 }
 
+type ChatTLFResolveArg struct {
+	Uid    keybase1.UID   `codec:"uid" json:"uid"`
+	ConvID ConversationID `codec:"convID" json:"convID"`
```

Comment:
Probably, but we don't have that from kbfstlfd yet. Will almost definitely add this when attacking sbs conflicts. For now the frontend could just refresh and the existing naming stuff takes care of it I think.